File: Python Programming/problems on bitwise operator/problem6.py
'''
6.Write a program which accept one number from user and off 7th bit of that number if it is on. 
	Return modified number. 
	
	Input :  79 
	Output : 15 
	
	Author : Milind Nalawade
	
	Date : 01/08/2021
	
	Language : Python 3.9.0

'''
#====
#Logic
#====
def Chkbit(no):
	mask=0x00000040
	if mask&no==mask:
		return mask^no
	return no	
	
#=============
#Entry point function
#=============
def main():
	no=int(input("\n Enter your number : "))
	ret=int(Chkbit(no))
	print("\n Output : {} ".format(ret))
	
#=====
#starter
#=====
if __name__== "__main__":
	main()
	

File: C++ Programming/problems on bitwise operator/problem6.cpp
/*

6.Write a program which accept one number from user and off 7th bit of that number if it is on. 
	Return modified number. 
	
	Input :  79 
	Output : 15 
	
	Author : Milind Nalawade
	
	Date : 02/08/2021
	
	Language : C++ programming 

*/
using namespace std;
#include<iostream>
//Logic
int ChkBits(int ino)
{
	int pos=0x00000040;
	int flag=pos&ino;
	if(flag==pos)
	{
		return ino^pos;
	}
	return ino;
}

//Entry point function
int main()
{
	int no=0,ret=0;
	cout<<"\n Enter the number :";
	cin>>no;
	ret=ChkBits(no);
	cout<<"\n Output : "<<ret<<"\n";
	return 0;
}	